homeassistant/components/bring/quality_scale.yaml

```
@@ -0,0 +1,70 @@
+rules:
+  # Bronze
+  action-setup:
+    status: todo
+    comment: The integration registers an entity service. Rule is unclear about this.
+  appropriate-polling: done
+  brands: done
+  common-modules: done
+  config-flow-test-coverage: done
+  config-flow: todo
+  dependency-transparency: done
+  docs-actions: done
+  docs-high-level-description: todo
+  docs-installation-instructions: todo
+  docs-removal-instructions: todo
+  entity-event-setup:
+    status: exempt
+    comment: The integration registers no events
+  entity-unique-id: done
+  has-entity-name: done
+  runtime-data: done
+  test-before-configure: done
+  test-before-setup: done
+  unique-config-entry: done
+
+  # Silver
+  action-exceptions: done
+  config-entry-unloading: done
+  docs-configuration-parameters: todo
+  docs-installation-parameters: todo
+  entity-unavailable: done
+  integration-owner: done
+  log-when-unavailable: done
```

```suggestion
  log-when-unavailable: 
      status: done
      comment: handled by coordinator
```
might need formatting